start/sw/web-app/laravel5.md

```diff
@@ -57,7 +65,7 @@ $ laravel new ProjectName
 ```bash
 $ cd ProjectName
 $ cp .env.example .env
-$ nano .env
+$ gedit .env
```

Comment:
Why would you change it? It seems a bit weird to change the editor. Nano is pretty standard for editing commands in tutorials.